src/Pages/Profile/index.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from 'react';
+import Dashboard from "../Dashboard"
+import { useDispatch, useSelector } from 'react-redux';
+import { getProfile } from '../../redux/actions/profile';
+
+const Profile = () => {
+
+    const dispatch = useDispatch()
+    const profile = useSelector(state => state?.profile?.data?.Profile)
```

Comment:
Here the state should be like state.profile only from redux you need to maintain deep level variable